app/Services/Validation/PasswordValidator.php

```diff
@@ -15,7 +15,7 @@ public function rules()
     {
         return [
             'password' => [
-                'min:8',
+                'min:9',
                 'regex:/^.*(?=.{3,})(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%^&*]).*$/',
```

Comment:
We should replace the `regex` rule here with `PasswordFormatRule` to keep it consistent with the entire site. Also, I don't think this validator is being referenced anywhere, now that I'm looking. If that's the case you can just delete the file.